app/assets/stylesheets/application.css

```diff
@@ -15,3 +15,20 @@
  *= require foundation_and_overrides
 
  */
+
```

Comment:
Any custom styling should be added to a `.css` or .scss` file rather than changing this manifest. Adding styling here should be considered an absolute last resort - organizing our files this way makes it much easier to hunt down where we defined a certain style attribute by having them all in a CSS file

Reply:
Moved the styling to a new file. Having troulbe using Scss global variables so they are at the top of both files